Description: bad aligned access considerations on ARM
 This patch is related with bug report #217221:
   SDLNet_(Read|Write)(16|32) assume host endianness is always LE)
 and the endian.patch
Author: Sam Hocevar <sam@zoy.org>
Last-Update: 2003-09-24
Bug-Debian: http://bugs.debian.org/212570

--- a/SDL_net.h
+++ b/SDL_net.h
@@ -354,7 +354,7 @@
 /* Inline macro functions to read/write network data */
 
 /* Warning, some systems have data access alignment restrictions */
-#if defined(sparc) || defined(mips)
+#if defined(sparc) || defined(mips) || defined(__arm__)
 #define SDL_DATA_ALIGNED	1
 #endif
 #ifndef SDL_DATA_ALIGNED
